akka-stream-tests/src/test/scala/akka/stream/io/TcpSpec.scala

```diff
@@ -430,7 +430,7 @@ class TcpSpec extends StreamSpec("akka.stream.materializer.subscription-timeout.
           .run()
 
       // make sure that the server has bound to the socket
-      val binding = Await.result(bindingFuture, 100.millis)
+      val binding = Await.result(bindingFuture, 1.second)
```

Comment:
Mostly because I don't know where/how that's applicable/good to use, everywhere?